### PR TITLE
#49 update request body and response body use contract API v1

### LINKED ISSUE #49: [BUGFIX] FIx request and response body using Contract API v1 Sapawarga
## Issue
Client need request body and response body the same as contract api v1 sapawarga app

## Tasks
- [x] update any request of video service like contract api v1
- [x] update any response of video service like contract api v1

### PR DESCRIPTION
**Pull request by setiadijoe**
close #49 
- update request body and response body
- update query for get statistic video

### CODE REVIEW

File: endpoint/endpoint.go [hunk 1]
```diff
@@ -38,36 +35,61 @@ func MakeGetDetailVideo(ctx context.Context, fs usecase.UsecaseI) endpoint.Endpo
...
-			CreatedBy:   resp.CreatedBy,
-			UpdatedBy:   resp.UpdatedBy,
-		}, nil
+		data := &VideoDetail{
+			ID:                 resp.ID,
+			Title:              resp.Title,
+			TotalLikes:         resp.TotalLikes,
+			IsPushNotification: resp.IsPushNotification,
+			Sequence:           resp.Sequence,
+			Cateogry:           resp.Category,
```
Comment (rindibudiaramdhan) [inline endpoint/endpoint.go:44]: typo I think

**Review by rindibudiaramdhan — CHANGES_REQUESTED** [review]

File: endpoint/endpoint.go (hunk 1 above)
Comment (setiadijoe) [reply endpoint/endpoint.go:44]: ooh thanks, I'll fix it

File: endpoint/endpoint.go [hunk 2]
```diff
@@ -38,36 +35,61 @@ func MakeGetDetailVideo(ctx context.Context, fs usecase.UsecaseI) endpoint.Endpo
...
 	return func(ctx context.Context, request interface{}) (response interface{}, err error) {
 		resp, err := fs.GetStatisticVideo(ctx)
 		if err != nil {
 			return nil, err
 		}
 
-		return &VideoStatisticResponse{
-			Data: resp,
+		meta := &model.Metadata{
+			Page:        20,
```
Comment (aldidoanta) [inline endpoint/endpoint.go:76]: @setiadijoe Is `20` an arbitrary value? Should we define a constant?

**Review by firmanJS — APPROVED** [review]

**Review by aldidoanta — APPROVED** [review]

File: endpoint/endpoint.go (hunk 2 above)
Comment (setiadijoe) [reply endpoint/endpoint.go:76]: ooh thanks, I think I should use constant

**Review by rindibudiaramdhan — APPROVED** [review]
go

**Comment by setiadijoe** [comment]
> go

SAFE TO MERGE ??
@aldidoanta @firmanJS @rindibudiaramdhan

**Pull request merged**